src/main/java/Assignment1.java

```diff
@@ -0,0 +1,175 @@
+import java.util.Scanner;
+
+class ItemList {
+    String _name, _type;
+    double _price;
+    int _quantity;
+
+    // method to extract word from a sentence from pos till end or '-' appears
+    String extractWord(String input_string, int pos) {
```

Comment:
follow variable naming convention (Ex : input_string should be written as inputString)

Reply:
done.